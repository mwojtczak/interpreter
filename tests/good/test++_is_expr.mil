function main () {
	a = 4;
	y = a++ + 1;
	print(a);
	print(y);
	y = a-- - 1;
	print(a);
	print(y);

	a = 10;
	y = --a - 1;
	print(a);
	print(y);

	a = 10;
	y = ++a + 1;
	print(a);
	print(y);
}
