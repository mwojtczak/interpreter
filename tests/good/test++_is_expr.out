5
5
4
4
9
8
11
12
